### PR TITLE
new changes

### PR DESCRIPTION
**Pull request by muskan7898**
new line

### CODE REVIEW

**Review by CipherKode — CHANGES_REQUESTED** [review]
Add some code in the pull request

File: main.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class Main {
+  public static void main(String[] args) {
+    System.out.println("Hello, World!");
+  }
```
Comment (CipherKode) [inline main.java:19]: Add a function to print the first n fibonacci numbers

**Review by CipherKode — CHANGES_REQUESTED** [review]
Add some code in the pull request

File: main.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+public class Main {
+  
+  public static void Printfibonacci(int n){
+    int a=0;
+    int b=1;
+    int c;
+    for(int i=0;i<n;i++){
+      System.out.println(a);
```
Comment (CipherKode) [inline main.java:8]: Print in the same line

**Review by CipherKode — CHANGES_REQUESTED** [review]

File: main.java (hunk 1 above)
Comment (muskan7898) [reply main.java:19]: done

**Review by CipherKode — APPROVED** [review]

**Pull request merged**